grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -490,7 +490,7 @@ private void close(Status status, Metadata metadata) {
         }
 
         final RequestLogBuilder logBuilder = ctx.logBuilder();
-        logBuilder.responseContent(GrpcLogUtil.rpcResponse(status, firstResponse), null);
+        logBuilder.responseContent(GrpcLogUtil.rpcResponse(status, firstResponse, metadata), null);
```

Comment:
It should be helpful if we add tests for the client side as well.

Reply:
Added https://github.com/line/armeria/pull/4636/commits/305be3d09bd7af1ab8374f1fc05908b262ac5187.